newIDE/app/src/EventsFunctionsExtensionEditor/OptionsEditorDialog/ExtensionOptionsEditor.js

```diff
@@ -190,6 +191,37 @@ export const ExtensionOptionsEditor = ({
             }}
             fullWidth
           />
+          <SemiControlledAutoComplete
+            floatingLabelText={<Trans>Category (shown in the editor)</Trans>}
+            fullWidth
+            value={eventsFunctionsExtension.getCategory()}
+            onChange={category => {
+              eventsFunctionsExtension.setCategory(category);
+              forceUpdate();
+            }}
+            dataSource={[
```

Comment:
did I also see 'network' and 'device' as categories?

Reply:
Great catch!